packages/admin-ui/client/pages/Item/index.js

```diff
@@ -77,7 +77,7 @@ const ItemDetails = withRouter(
           }
           history.push(`${adminPath}/${list.path}`);
 
-          toastItemSuccess(this.props.toast, item, 'Deleted successfully');
+          toastItemSuccess(this.props.toastManager, item, 'Deleted successfully');
         })
         .catch(error => {
           toastError(this.props.toast, error);
```

Comment:
```suggestion
           toastError(this.props.toastManager, error);
```

Reply:
This didn't fix the issue, I am investigating further

Reply:
Hmm I've been looking into it, and it looks like `toastManager` is not being passed to `toastSuccess` when deleting an item.

In `toastItemSuccess`, `toast` is undefined.

Reply:
I have a temporary fix which just creates toast content similar to `onSave`

Reply:
I think part of the problem might be here since I don't think .addToast is actually a thing in react-toast-notifications
https://github.com/keystonejs/keystone-5/blob/a15601ee6b551eae04f41289c339dbef8315843a/packages/admin-ui/client/util.js#L39-L42
I think this might fix it (along with passing in `this.props.toastManager` + applying the same fix for the error state)
```jsx
toast.add(toastContent, { 
  autoDismiss: true, 
  appearance: 'success', 
}); 
```

Reply:
Ah awesome, i arrived at the same thing. I haven't used react-toast-notifications before, but am guessing that if this worked properly before, the API must have changed?
By changing it to `toast.add` and not immediately invoking the function, everything goes back to working as expected.

Reply:
My bad, sorry folks --> I updated the API.